shadow-dom/untriaged/events/events-that-are-always-stopped/test-001.html

```diff
@@ -42,12 +42,12 @@
     s.appendChild(inp1);
 
     s.addEventListener('abort', A_05_04_01_T01.step_func(function(event) {
-    	assert_equals(event.target.getAttribute('id'), 'inp1', 'Inside shadoe tree: Wrong target');
+    	assert_equals(event.target.getAttribute('id'), 'inp1', 'Inside shadow tree: Wrong target');
     }), false);
 
     d.body.addEventListener('abort', A_05_04_01_T01.step_func(function(event) {
-    	assert_true(false, 'Abort event should always be stopped at Shadow boundary');
-    }), false);
+        assert_true(true, 'Abort event should not be stopped at Shadow boundary if created by users');
+    }), true);
```

Comment:
Originally this was a test that if 'abort' event bubbles on <body>, this assertion checks that it is an
unexpected behavior.
With this change, now 'abort' event bubbles on <body> and it passes, but it can't detect the event
does NOT bubble on <body>.